docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -209,19 +209,19 @@ In the two examples below you see the outline of how the `dlt.sources.incrementa
 def tweets():
 	# Get a last value from loaded metadata. If not exist, get None
 	last_val = dlt.current.state().setdefault("last_updated", None)
-	# get data and yield it
-	data = get_data(start_from=last_val)
-	yield data
-	# change the state to the new value
+  # get data and yield it
+  data = get_data(start_from=last_val)
+  yield data
+  # change the state to the new value
   dlt.current.state()["last_updated"]  = data["last_timestamp"]
 ```
 if we use a list or a dictionary, we can modify the underlying values in the objects and thus we do not need to set the state back explicitly.
 ```python
 @resource()
 def tweets():
-	# Get a last value from loaded metadata. If not exist, get None
-	loaded_dates = dlt.current.state().setdefault("days_loaded", [])
-	# do stuff: get data and add new values to the list
+  # Get a last value from loaded metadata. If not exist, get None
+  loaded_dates = dlt.current.state().setdefault("days_loaded", [])
+  # do stuff: get data and add new values to the list
   # `loaded_date` is a shallow copy of the `dlt.current.state()["days_loaded"]` list
   # and thus modifying it modifies the state
   yield data
```

Comment:
the line `loaded_dates.append('2023-01-01')` should also be edited

Reply:
it has two spaces like it should? I will merge and we can fix it later if needed